index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {

```

Comment:
You should export a javascript class. See http://www.phpied.com/3-ways-to-define-a-javascript-class/

```javascript
module.exports = Jawn

function Jawn () {
}

Jawn.prototype.import = function() { ... }

```

Similar to how hypercore defines its Hypercore class https://github.com/mafintosh/hypercore/blob/master/index.js#L11-L13